projects/igniteui-angular/src/lib/date-picker/date-picker.component.ts

```diff
@@ -37,7 +37,8 @@ import {
     VerticalAlignment,
     HorizontalAlignment,
     PositionSettings,
-    ConnectedPositioningStrategy
+    ConnectedPositioningStrategy,
+    AbsoluteScrollStrategy
```

Comment:
Shouldn't we import the scrolling strategy from the exact file where it's defined? `../services/overlay/scroll/absolute-scroll-strategy`

Reply:
Both are fine.